hack/go-fmt.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+REPO_NAME=$(basename "${PWD}")
+if [ "$IS_CONTAINER" != "" ]; then
+  for TARGET in "${@}"; do
+    find "${TARGET}" -name '*.go' ! -path '*/vendor/*' ! -path '*/.build/*' -exec gofmt -s -w {} \+
```

Comment:
It would be better to pass this through xargs. There may be a lot of .go files.

Reply:
> It would be better to pass this through xargs. There may be a lot of .go files.

Scrap this - presumably we're only running it across a subset (i.e., openshiftmachineapi). :)

Reply:
These are copied and pasted mostly from one of our other repos.